avocado/plugins/multiplex.py

```diff
@@ -35,12 +35,10 @@ def __init__(self, *args, **kwargs):
         super(Multiplex, self).__init__(*args, **kwargs)
 
     def configure(self, parser):
-        if multiplexer.MULTIPLEX_CAPABLE is False:
-            return
-
         parser = super(Multiplex, self).configure(parser)
-        parser.add_argument('multiplex', nargs='+',
-                            help='Path(s) to a multiplex file(s)')
+        if multiplexer.MULTIPLEX_CAPABLE:
+            parser.add_argument("-m", '--multiplex', nargs='*',
```

Comment:
Yes and as I said it's not supported (not even for plugins, so each time it could be different). For simple cases it'd work well, for advanced ones you can always rewrite your configs and use only one implementation (which is what a sane person would do).

I don't expect people to mix and match different types, my point was that it's by design how the Mux database works. The only defined order is `--mux-inject` first, then defaults like `avocado-virt` and last the plugins (in undefined order).